src/main/webapp/css/home.css

```diff
@@ -18,7 +18,7 @@
 }
 
 .search-bar {
-  background-color: #f7f7f7;
+  outline: 2px solid #ededed;
```

Comment:
Is there a reason not to use the "card" class in Bootstrap to get the gray outline?

Reply:
Yeah I tried using that class but it messed up the formatting of the components inside